### PR TITLE
Update Client.php

### PR DESCRIPTION
**Pull request by jrean**
Docblock.

### CODE REVIEW

**Review by derrabus — APPROVED** [review]

File: src/Symfony/Component/BrowserKit/Client.php [hunk 1]
```diff
@@ -365,7 +365,7 @@ protected function doRequestInProcess($request)
             unlink($deprecationsFile);
             foreach ($deprecations ? unserialize($deprecations) : array() as $deprecation) {
                 if ($deprecation[0]) {
-                    trigger_error($deprecation[1], E_USER_DEPRECATED);
+                    @trigger_error($deprecation[1], E_USER_DEPRECATED);
```
Comment (fabpot) [inline src/Symfony/Component/BrowserKit/Client.php]: This one should be reverted

File: src/Symfony/Component/BrowserKit/Client.php [hunk 2]
```diff
@@ -606,6 +606,14 @@ protected function requestFromRequest(Request $request, $changeHistory = true)
         return $this->request($request->getMethod(), $request->getUri(), $request->getParameters(), $request->getFiles(), $request->getServer(), $request->getContent(), $changeHistory);
     }
 
+    /**
+     * Update the server setting from the given URI.
```
Comment (fabpot) [inline src/Symfony/Component/BrowserKit/Client.php]: `Updates`

File: src/Symfony/Component/BrowserKit/Client.php [hunk 3]
```diff
@@ -616,6 +624,13 @@ private function updateServerFromUri($server, $uri)
         return $server;
     }
 
+    /**
+     * Extract host value from the URI.
```
Comment (fabpot) [inline src/Symfony/Component/BrowserKit/Client.php]: `Extracts`

File: src/Symfony/Component/BrowserKit/Client.php (hunk 2 above)
Comment (fabpot) [reply src/Symfony/Component/BrowserKit/Client.php]: Oh, that's a private method, we don't document private method except when really needed. So, this docblock and the next should be removed.

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/BrowserKit/Client.php (hunk 1 above)
Comment (jrean) [reply src/Symfony/Component/BrowserKit/Client.php]: I reverted it (your fabbot was asking to add the `@`)

File: src/Symfony/Component/BrowserKit/Client.php (hunk 2 above)
Comment (jrean) [reply src/Symfony/Component/BrowserKit/Client.php]: I didn't notice! Sorry. I reverted.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @jrean.

**Pull request closed without merging**